releasenotes/notes/deprecate-stabilizer-table-9efd08c7de1a5b4d.yaml

```diff
@@ -0,0 +1,10 @@
+---
+deprecations:
+  - |
+    The :class:`~qiskit.quantum_info.PauliList` is deprecated and will be removed in a future
```

Comment:
should be: :class:`~qiskit.quantum_info.PauliTable` is deprecated

Reply:
My bad. Thank you.

Reply:
StabilizerTable is deprecated here